string direction
string slewRate
---
TelescopeInfo status
